### PR TITLE
non standard eval gotcha fix

### PR DESCRIPTION
**Pull request by jread-usgs**
and #103

### CODE REVIEW

**Comment by jread-usgs** [comment]
@aappling-usgs take a quick look?

File: R/stage_nldas_ts.R [hunk 1]
```diff
@@ -46,7 +46,7 @@ stage_nldas_ts <- function(sites, var, times, folder = tempdir(), verbose = FALS
   DateTime <- matches <- variable <- ".dplyr.var"
   for (i in 1:length(sites)){
 
-    site_data <- select(data_out, DateTime, matches(sites[i]), variable, units) %>%
+    site_data <- select_(data_out,'DateTime',sites[i],'variable','units') %>%
```
Comment (aappling-usgs) [inline R/stage_nldas_ts.R]: sites[i] is the exact column name, right? not just part of it?

File: R/stage_nldas_ts.R [hunk 2]
```diff
@@ -56,8 +56,11 @@ stage_nldas_ts <- function(sites, var, times, folder = tempdir(), verbose = FALS
       setNames(c('DateTime',var)) %>%
       u(c(NA, units))
     
-    fpath <- write_ts(site_data, site=sites[i], var=var, src="nldas", folder)
-    file_paths <- c(file_paths, fpath)
+    if (!any(is.na(site_data[var]))){
```
Comment (aappling-usgs) [inline R/stage_nldas_ts.R]: sorta stringent test here - will site_data ever have just a few NAs such that we'll actually want to keep it?

Comment (jread-usgs) [inline R/stage_nldas_ts.R]: good point, will change to `!all()`

File: R/stage_nldas_ts.R (hunk 1 above)
Comment (jread-usgs) [inline R/stage_nldas_ts.R]: yes

Comment (aappling-usgs) [inline R/stage_nldas_ts.R]: then looks ducky to me :bird:

**Comment by aappling-usgs** [comment]
tests are no worse than they were before - testthat error i can't reproduce on my machine, plus non-standard travis and appveyor files.

**Pull request merged**